cmd/agent-js-worker/main.go

```diff
@@ -839,7 +842,18 @@ func prepareFormattedProvider(kmsStorageProvider storage.Provider, kmsImpl kms.K
 
 	encryptedFormatter := edv.NewEncryptedFormatter(jweEncrypter, jweDecrypter)
 
-	return formattedstore.NewFormattedProvider(provider, encryptedFormatter, false), nil
+	var opts []formattedstore.Option
+
+	if edvCache {
+		p, err := jsindexeddb.NewProvider("cache")
```

Comment:
You will need to expire and clear the cache for this to work when switching between browsers.

Reply:
done